.devcontainer/devcontainer.json

```diff
@@ -51,7 +51,9 @@
 				"ms-python.isort",
 				"ms-python.vscode-pylance",
 				"ms-vscode.makefile-tools",
-				"matangover.mypy"
+				"matangover.mypy",
+				"ms-python.mypy-type-checker",
+				"charliermarsh.ruff"
```

Comment:
Feel free to ignore this, but with the addition of ruff I am pretty sure you can remove flake, black and isort above. So if you dont mind you can try and if it works then clean up the extensions.

Reply:
Happy to do this. I didn't do it initially because I didn't know if it would disrupt other people's workflows.

Reply:
Ok yeah up to you. As you suggest though it is probably better to not risk messing with the configuration now and messing up peoples workflows.